crates/bevy_app/src/plugin_group.rs

```diff
@@ -23,6 +23,24 @@ pub struct PluginGroupBuilder {
 }
 
 impl PluginGroupBuilder {
+    /// Finds the index of a target [`Plugin`]. Panics if the target's [`TypeId`] is not found.
+    fn index_of<Target: Plugin>(&mut self) -> usize {
+        let index = self
+            .order
+            .iter()
+            .enumerate()
+            .find(|(_i, ty)| **ty == TypeId::of::<Target>())
```

Comment:
Could we use [`position`](https://doc.rust-lang.org/std/iter/struct.Enumerate.html#method.position) here instead of the `find`?

Reply:
good point. `position` would be a better fit in this situation. removes both the enumerate and map calls.